tests/asset-css.test.js

```diff
@@ -234,6 +234,11 @@ tap.test(
             rel: 'stylesheet',
         });
 
+        t.equal(
+            obj.toEarlyHint(),
+            '</foo>; crossorigin="bar"; type="text/css"; rel="stylesheet";',
```

Comment:
[The examples on MDN are without quotes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/103#preload_example)